### PR TITLE
chore (DDI-1271): Update container docs

### PR DESCRIPTION
**Pull request by amar-ABgov**

### CODE REVIEW

File: libs/docs/src/components/common/Container.stories.mdx [hunk 1]
```diff
@@ -357,19 +357,35 @@ export const HeadingTemplate = (args) => {
...
     <GoAGrid minChildWidth="200px">
-      <GoAContainer accent="thick" title="Heading">
+      <GoAContainer accent="thick" title={<div>Group Heading</div>}>
         Content
       </GoAContainer>
       <GoAContainer
         type="non-interactive"
         accent="thick"
-        title="Heading"
+        title={<div>Group Heading</div>}
```
Comment (chrisolsen) [inline libs/docs/src/components/common/Container.stories.mdx:372]: The `title` attribute will accept a string, so it just a string so this example will be misleading.

Comment (chrisolsen) [reply libs/docs/src/components/common/Container.stories.mdx:372]: However, an example should be included that contains a more complex header that will utilize the ReactNode/slot change that was made.

**Comment by Radha-Alberta** [comment]
- The highlighted example (from storybook doc) is giving error when run in angular demo app.

image
image

- The title "The container title" is missing 
image
image

**Review by chrisolsen — DISMISSED** [review]

**Review by ArakTaiRoth — DISMISSED** [review]

**Review by ArakTaiRoth — APPROVED** [review]

**Review by chrisolsen — APPROVED** [review]

**Pull request merged**